tests/test_builtin_casters.cpp

```diff
@@ -72,6 +72,11 @@ TEST_SUBMODULE(builtin_casters, m) {
     m.def("string_view32_return", []() { return std::u32string_view(U"utf32 secret \U0001f382"); });
 #endif
 
+    m.def("i32_str", [](std::int32_t v) { return std::to_string(v); });
```

Comment:
Can you add a comment above this line to associate it with the Python test function:
```c++
// test_integer_casting
```